lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -583,6 +584,13 @@ module.exports = class MethodMappings {
           return value;
         },
 
+        async isElementPresent(webElementOrId) {
+          const element = this.getWebElement(webElementOrId);
+
+          return element instanceof WebElement || element instanceof ShadowRoot;
+        },
```

Comment:
Did you test the command yourself by using it in some example tests. I feel that this would not work in case the element does not exist.